exchanges/binanceus/binanceus_websocket.go

```diff
@@ -46,9 +46,9 @@ var (
 // WsConnect initiates a websocket connection
 func (bi *Binanceus) WsConnect() error {
 	if !bi.Websocket.IsEnabled() || !bi.IsEnabled() {
-		return stream.ErrWebsocketNotEnabled
+		return websocket.ErrWebsocketNotEnabled
 	}
-	var dialer websocket.Dialer
+	var dialer gws.Dialer
```

Comment:
Suggestion: Can have most of the gws vars shadowed in our websocket package, like `Dialer`, `PongMessage` . So that way we just always point to websocket and not have to worry.

Reply:
I think that's not going to work whilst `.Conn` is a real gorilla Connection and expects a concrete type of `*gws.Dialer` in `gws.Conn.Dial()`

I think we can abstract away the need to do this, or call Dial on the conn directly, but that's going to be part of later work, not part of this restructuring.

Resolving.